pdfparser.d.ts

```diff
@@ -0,0 +1,130 @@
+import {EventEmitter} from "events";
+
+declare class Pdfparser extends EventEmitter{
+    constructor();
+    parseBuffer(buffer: Buffer): void;
+    loadPDF(pdfFilePath: string, verbosity?: number):Promise<void>
+    createParserStream():ParserStream
+    on<K extends keyof EventMap>(eventName: K, listener: EventMap[K]): this
+}
+
+type EventMap = {
+    "pdfParser_dataError": (errMsg: string) => void;
+    "pdfParser_dataReady": (pdfData: Output) => void;
+    "readable": (meta: Output["Meta"]) => void;
+    "data": (data: Output["Pages"][number]|null) => void;
+}
+
+declare class ParserStream{
+    //TODO
```

Comment:
should be quick to fill it in

Reply:
The thing is that i don't understand how this work.